packages/configs/README.md

```diff
@@ -0,0 +1,66 @@
+# Overview
+
+Base Configuration files for platform (example `@aws-amplify/ui-vue`) and utility packages (example `@aws-amplify/ui-react-core`) in the Amplify UI monorepo.
+
+## Package Types
+
+There are currently 3 types of packages that Amplify UI supports:
+
+- **Utility Packages:** Packages shared across all platforms
+
+  - `@aws-amplify/ui`
+
+- **React Utility Packages:** Non-UI React utilites and hooks shared between React and React Native platform packages
+
+  - `@aws-amplify/ui-react-core`
+
+- **Platform UI Packages:**
+  - `@aws-amplify/ui-angular`
+  - `@aws-amplify/ui-react`
+  - `@aws-amplify/ui-react-native`
+  - `@aws-amplify/ui-vue`
+
+## Typescript
+
+To prevent drift between the `compilerOptions` of the _tsconfig.json_ files in the Amplify UI monorepo as more packages are added, base Typescript `compilerOptions` configurations are stored within the [ts](./ts) directory. The **[Base Config](./ts/base.json)** is extended directly or indirectly in all platform and utility packages.
+
+- Utility package _tsconfig.json_ files directly use the **[Base Config](./ts/base.json)**
+- React UI/React Utility package _tsconfig.json_ files use the **[React Base Config](./ts/react.json)**, extends from the **Base Config**
+- React Native UI package _tsconfig.json_ files use the **[React Native Base Config](./ts/react-native.json)**, extends from the **Base Config**
+
+A basic visualization of the extension hierarchy:
+
+```tree
+base.json
+├── react.json
+└── react-native.json
+```
+
+### Strict Mode
+
+All packages are in TS strict mode by default unless opted out at the package configuration level.
+
+### Usage of _tsconfig.dist.json_
```

Comment:
What do you think about naming this to  `_tsconfig.prod.json_` so that this is more explicit?

Reply:
Not feeling super strongly (so basically feeling minimally strong) about the name, but prefer _tsconfig.dist.json_ because:
- we already use it
- it aligns with the _dist_ output directory